web-components/src/rootStyles/design-tokens.mdx

```diff
@@ -50,16 +49,31 @@ properties.
 
 ## Typography
 
-<Canvas>
```

Comment:
Canvas now always has to reference a story, but creating a Typeface story will also show the story in the sidebar. The `Typedef` Storyblock does not handle the fonts as we used to show it so I set up a little structure that shows the typefaces nicely
<img width="1052" alt="Screenshot 2024-06-09 at 22 23 09" src="https://github.com/Zuehlke/design-system-starter/assets/1119248/804ec5ec-8615-4b75-a94c-a1d2ab646426">